framework/src/Volo.Abp.EventBus.RabbitMQ/Volo/Abp/EventBus/RabbitMq/AbpRabbitMqEventBusOptions.cs

```diff
@@ -2,24 +2,6 @@
 
 namespace Volo.Abp.EventBus.RabbitMq;
 
-public class AbpRabbitMqEventBusOptions
+public class AbpRabbitMqEventBusOptions : AbpRabbitMqExchangeAndQueueBaseOptions
```

Comment:
I think we don't need to add `AbpRabbitMqExchangeAndQueueBaseOptions`.

Because it is designed for the EventBus.